src/lib/decisionsCallbackStorage.js

```diff
@@ -0,0 +1,12 @@
+const decisionsReceivedEventTriggers = [];
```

Comment:
add copyright and to a couple other files.

Reply:
yes thank you